controllers/user.controller.js

```diff
@@ -0,0 +1,48 @@
+const User = require("../models/user.model");
+
+exports.create = function(req, res){
+    let user = new User(
+        {
+            username: req.body.username,
+            password: req.body.password
```

Comment:
I'm not to sure if we will have a password field. We can leave it for now, but we are doing authentication with firebase, so we might need a firebase id or something whenever a user is getting created. We'll have to make a poc for that entire user story (make account on frontend -> firebase -> nodejs service).